src/Microsoft.AspNet.Server.Kestrel/Infrastructure/TaskUtilities.cs

```diff
@@ -12,5 +15,72 @@ public static class TaskUtilities
 #else
         public static Task CompletedTask = Task.FromResult<object>(null);
 #endif
+
+        private static WaitCallback _runAction = (o) =>
+        {
+            try
+            {
+                ((Action)o)();
+            }
+            catch (Exception)
+            {
+                // log with a static logger in some way?
+            }
+        };
+
+        private static WaitCallback _completeTcs = (o) =>
+        {
+            try
+            {
+                ((TaskCompletionSource<object>)o).SetResult(null);
```

Comment:
No, and I doubt it ever would as this is the only place that ever sets it and it is called only once - however not catching an exception at the top level of a threadpool thread; which is at the top of the stack, will crash the process (by design) as its kinda like `async void`